src/lib/components/consignments/__stories__/index.tsx

```diff
@@ -1,21 +1,20 @@
 import "./consignments.story"
-import "./typography.story"
+import "./style.story"
 
 import { storiesOf } from "@storybook/react-native"
 import * as React from "react"
 import { View } from "react-native"
 
-import * as bottomAlignedButton from "./consignments-bottom-aligned.story"
```

Comment:
Moved all of my kebab named files to lowercase, this is mainly to be consistent with code that came before me - we should consider adding a danger rule for this ( thinking of the same for Jest too https://github.com/facebook/jest/issues/3771 )